test/index.test.ts

```diff
@@ -289,7 +297,7 @@ describe('Probot', () => {
       const redisConfig = {
         host: 'test'
       }
-      const probot = createProbot({ webhookPath: '/webhook', githubToken: 'faketoken', redisConfig})
+      probot = createProbot({ webhookPath: '/webhook', githubToken: 'faketoken', redisConfig })
 
       expect(probot.throttleOptions.Bottleneck).toBe(Bottleneck)
```

Comment:
Property 'throttleOptions' is private and only accessible within class 'Probot'.